Gemfile

```diff
@@ -1,9 +1,9 @@
 # Generated from /Users/light/work/my/acts_as_commentable_with_threading/acts_as_commentable_with_threading.gemspec
 source 'http://rubygems.org'
 
-gem "activerecord", "~> 4.0"
-gem "activesupport", "~> 4.0"
-gem "awesome_nested_set", "~> 3.0.0.rc.2"
+gem "activerecord", ">= 5.2"
+gem "activesupport", ">= 5.2"
```

Comment:
As per the existing gemspec, the minimal version of Rails referenced is Rails 5.2.